builtins/amp-img.js

```diff
@@ -63,6 +69,13 @@ export class AmpImg extends BaseElement {
     }
   }
 
+  /** @override */
+  onMeasureChanged() {
+    if (isExperimentOn(this.getAmpDoc().win, 'amp-img-auto-sizes')) {
```

Comment:
Store this in a private property in constructor.

Reply:
Is it ok use `this.getAmpDoc()` in the constructor? I've noticed that people generally put this line of code in `buildCallback`, but `<amp-img>` doesn't use `buildCallback`.